### DISCUSSION TITLE
How does the table get updated?

### DISCUSSION THREAD

**Discussion by qazwsxedckll** [sadmann7/tablecn#794] [Q&A] [answered]
Please help. I am so confused. When I click delete, how does the table get updated without redirect or refresh? Is the `revalidateTag` doing the trick? The doc says `revalidateTag only invalidates the cache when the path is next visited. `

I have built a simple table, when I deleted the row in the database, I need to refresh the page to get the table updated.

**Comment by sadmann7** [comment, marked as answer]
yeah its revalidateTag.

the tasks query is cached with a key, just revalidating it on each mutation.

most likely you tagged the key wrong or something.

did you try `revalidatePath`? that should also work afaik.
